test/property.test.js

```diff
@@ -125,6 +207,20 @@ describe('loopback:property generator', function() {
     });
   });
 
+  it('should fail to create with unsupported type', function (done) {
+    var propertyGenerator = givenPropertyGenerator();
+    helpers.mockPrompt(propertyGenerator, {
+      model: 'Car',
+      name: 'something',
+      type: 'unknown',
```

Comment:
The programatic part only sets up process environment (cwd, yeoman environment, etc.). Everything else (inquirer prompts in particular) is executed the same way as if the generator was started directly via `yo loopback`.